boms/cloud-oss-bom/pom.xml

```diff
@@ -45,8 +45,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <guava.version>28.1-android</guava.version>
-    <google.cloud.java.version>0.116.0-alpha</google.cloud.java.version>
-    <io.grpc.version>1.24.1</io.grpc.version>
+    <google.cloud.java.version>0.119.0-alpha</google.cloud.java.version>
```

Comment:
I'm confused. You use this for `google-cloud-bom`, not `google-cloud-java`.

Reply:
google-cloud-bom is how we pull in google-cloud-java.